doc/write_your_own/components/intro.rst

```diff
@@ -1,29 +1,71 @@
 Cube Browser Components
 =======================
 
+Construction
+------------
+
+**Decide on options and widgets**
+
+To start making your Explorer, you need to decide which aspects of cube plotting you would like to offer options for and which aspects you would like to hard code.
+The examples in the notebooks below show some of the options that you might want to offer, and the widgets that you could use to display them.
+The syntax to create each widget is very similar, but some small aspects (like the widget type) differ.  The basic outline looks like this:::
+
+    widget1 = ipywidgets.ToggleButtons(
+              description='Dimension:',
+              options=coordinates,
+              value = 'time')
+
+For more detailed guidance and a more complete list of widgets that are available, please visit the ipywidgets website.
+
+**Construct one widget per cell**
+
+Once you have decided what options and widgets you would like to present you can treat them as your individual components, each possessing a Jupyter notebook cell.
+Each component can be displayed using this line at the bottom of the cell:::
+
+    IPython.display.display()
+
+**Make plotting button**
+
+At the end of your notebook you will need to add a cell which creates your plots using the values selected in the options widgets.  This will incorporate the Browser call.
+It is convenient to make a button to achieve this (see button example below) as it will be necessary at a later stage of Explorer development.
+
+Usage
+-----
+
+To use your Explorer, run a cell to produce your selection widget and make the selection for that widget before running the next cell.
+If you would like to change any of your options retrospectively, you will have to re-run the following widget after that change to ensure that the values update.
+This is important if you have any dependencies between widgets (see Options List for an example of this).
+
+
 Links to Notebooks:
 -------------------
 
-CoordSlider.link
+:download:`Value Catcher <ValueCatcher.ipynb>`
```

Comment:
I wonder whether all these links could do with being embedded into the usage section above rather than being separated out into this section. By doing this it would be clearer which example links to which bit of usage text - rather like including some static code in-line with the text.